src/frontend/src/utils/pipes.ts

```diff
@@ -170,3 +170,14 @@ export const meetingStartTimePipe = (times: number[]) => {
   const hours = ((time - minutes) / 60) % 12 === 0 ? 12 : ((time - minutes) / 60) % 12;
   return hours + (minutes !== 0 ? ':' + minutes : '') + (time >= NOON_IN_MINUTES ? 'pm' : 'am');
 };
+
+export const meetingDatePipe = (date?: Date) => {
```

Comment:
Can you add a comment to this, I think it'll be useful elsewhere but we should make sure to document it well